build/Dockerfile.s390x

```diff
@@ -20,7 +20,7 @@ RUN wget -O /qemu-s390x-static https://github.com/multiarch/qemu-user-static/rel
 
 RUN chmod +x /qemu-s390x-static
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1-398
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:f8174ab3c2fb4bd8aae29c48e077ba8e67c91904a639dd950b4c7e21c857cfec
```

Comment:
please use the s390x base image. this one is for ppc64le.

Reply:
revert back to registry.access.redhat.com/ubi8/ubi-minimal:8.1-398, correct?